tools/local-network/local_network.py

```diff
@@ -152,7 +152,7 @@ def __init__(self, name, node_num, client_port, peer_port, admin_port, admin_htt
         self.admin_http_gateway_port = admin_http_gateway_port
         self.peers = peers
         self.quorum_set = quorum_set
-        self.minimum_fee = 10_000_000_000
+        self.minimum_fee = 400_000_000
```

Comment:
thank you

i wonder if there is a more maintainable way to do this

is the issue that sample paykit is not detecting the fee dynamically? maybe that is a different bug?

Reply:
because like, we use the test client in prod to detect outages, and if it will break if the fee changes then that is bad

Reply:
nm, reading the code again, i see that this is working i guess

Reply:
I opened https://github.com/mobilecoinfoundation/mobilecoin/issues/1582 for this